src/styles/main.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import './blocks/resets';
+@import './blocks/variables';
+@import './blocks/calendar';
+@import './blocks/loops';
```

Comment:
good practice is to leave 1 line at the end of the file (please do this fix in each your file)